verified_boot_app_spec.md

```diff
@@ -0,0 +1,634 @@
+|   This system has been booted for the first time or has    |
+|   restored default settings. The wizard will assist in     |
+|   UEFI Secure Boot provisioning and trusted key database   |
+|   configuration.                                           |
+|                                                            |
+|   Boot0001:  Fedora (on SATA SSD #1 ESP)                   |
+|                                                            |
+|   Key Fingerprint (SHA-256):                               |
+|   AB34EF...                                                |
++------------------------------------------------------------+
+|   A new bootloader key has been detected.                  |
+|   Do you want to trust this key and continue booting?      |
++------------------------------------------------------------+
+| > Do NOT trust and proceed to the next key/bootloader      |
+| > Trust this key and Boot                                  |
+| > Trust this key and proceed to the next key/bootloader    |
+| > Show Key Details                                         |
++------------------------------------------------------------+
+| > Reboot                                                   |
++------------------------------------------------------------+
+|   Use ↑↓ to navigate.                                      |
+|   Use Enter to confirm selection.                          |
+|   Use ESC to go back to previous menu.                     |
++------------------------------------------------------------+
+```
+
+### 6.1.2 Key trust Prompt
+
+When user decides to trust the key, the application must confirm user's
+decision with second prompt:
+
++------------------------------------------------------------+
+|       Confirm Trust Decision for Key Fingerprint           |
++------------------------------------------------------------+
+|                                                            |
+|   Fingerprint (SHA-256):                                   |
+|   AB34EF...                                                |
+|                                                            |
+|   Are you sure you want to trust this key?                 |
+|   This will allow the system to boot and save the key.     |
+|                                                            |
++------------------------------------------------------------+
+|                                                            |
+| > Yes, Trust This Key                                      |
+| > No, Go Back                                              |
+|                                                            |
++------------------------------------------------------------+
+|   Use ↑↓ to navigate.                                      |
+|   Use Enter to confirm selection.                          |
+|   Use ESC to go back to previous menu.                     |
++------------------------------------------------------------+
+
```

Comment:
This mockup is missing Markdown code fences.

Reply:
Fixed: 164d282